tests/integration/test_charm.py

```diff
@@ -175,14 +175,24 @@ async def test_cluster_is_stable_after_deletion(ops_test: OpsTest) -> None:
     except RetryError:
         primary = None
 
-    client.close()
-
     # verify that the primary is not None
     assert primary is not None, "replica set with uri {} has no primary".format(replica_set_uri)
 
     # check that the primary is not one of the deleted units
     assert primary[0] in ip_addresses, "replica set primary is not one of the available units"
```

Comment:
This is not part of the change list, but I saw that the error message is not consistent with the check and the above comment.